dxr/static_unhashed/css/tree-selector.css

```diff
@@ -9,14 +9,13 @@
 
 .tree-selector {
     display: inline-block;
-    position: relative;
     float: right;
     width: auto;
+    text-align: right;
```

Comment:
I'm wondering if d.m.o's tree list is going to be harder to read with this - is it for a particular purpose?